mapillary_tools/process_geotag_properties.py

```diff
@@ -41,8 +41,7 @@ def process_geotag_properties(import_path,
                                                   "failed",
                                                   verbose)
         return
-
-    elif geotag_source != "exif" and not os.path.isfile(geotag_source_path):
+    elif geotag_source not in ["exif", 'blackvue'] and not os.path.isfile(geotag_source_path):
```

Comment:
did you change this just because blackvue geotag source is a dir and not file?
some check is needed, since if geo data is not in exif than it must be somewhere else and that location must exist